File: OOP_3A/PartII/Tnode.cpp
typedef class Tnode* TnodePtr;
class Tnode{
public:
    TnodePtr left;
    TnodePtr right;
    int val;
    Tnode(){
        left = nullptr;
        right = nullptr;
        val = 0;
    }
    Tnode(int item)
    {
        val = item;
        left = nullptr;
        right = nullptr;
    }
};

File: OOP_3A/PartII/Tnode.java
class Tnode{
    public Tnode left;
    public Tnode right;
    public int val;
    Tnode(){
        left = null;
        right = null;
        val = Integer.MIN_VALUE;
    }
    Tnode(int item)
    {
        val = item;
        left = null;
        right = null;
    }
}
